esx_service/utils/auth_data.py

```diff
@@ -667,12 +666,12 @@ def _remove_volumes_for_tenant(self, tenant_id):
                         return err
```

Comment:
If we hit error when deleting one volume, I think we should continue to delete other volumes. We should return error after go through all volumes.